dpe/src/commands/rotate_context.rs

```diff
@@ -47,7 +47,14 @@ impl<C: Crypto> CommandExecution<C> for RotateCtxCmd {
 
         // Make sure the command is coming from the right locality.
         if dpe.contexts[idx].locality != locality {
-            return Err(DpeErrorCode::InvalidHandle);
+            return Err(DpeErrorCode::InvalidLocality);
+        }
+        
+        // Make sure caller's locality does not already have a default context.
+        let default_context_idx =
+                dpe.get_active_context_pos(&ContextHandle::default(), locality);
+        if default_context_idx.is_some() {
+            return Err(DpeErrorCode::InvalidArgument);   
```

Comment:
We only want to do this check if the caller is trying to make this context a default context. The caller will make that type of request with the `TARGET_IS_DEFAULT` flag in the `flags` field.

If you want an example of how we have been parsing flags so far, the `DeriveChild` command in `dpe/src/commands/derive_child.rs` has a similar flag.

Reply:
Done.